Frontend/ui-driver/src/Screens/ReferralScreen/Controller.purs

```diff
@@ -286,4 +297,20 @@ transformLeaderBoard (DriversInfo driversInfo) isMaskedName = {
   , profileUrl : Nothing
   , rank : driversInfo.rank
   , rides : driversInfo.totalRides
-}
\ No newline at end of file
+}
+
+getReferralStage :: Merchant -> ReferralType
+getReferralStage merchant =
+  case merchant of
+    NAMMAYATRI -> LeaderBoard
+    YATRI -> QRScreen
```

Comment:
remove  `NAMMAYATRI`  it will be handled in default

Reply:
It good to handle different instead of relaying on default